File: Stage3/C/Problem11021.c
#include <stdio.h>

int main() {
    int T, A, B;
    scanf("%d", &T);
    int results[T];

    for (int i = 1; i <= T; i++) {
        scanf("%d %d", &A, &B);
        results[i] = A + B;
    }

    for (int i = 1; i <= T; i++) {
        printf("Case #%d: %d\n",i , results[i]);
    }

    return 0;
}

File: Stage3/Python/Problem11021.py
import sys
def main():
    input = sys.stdin.readline
    T = int(input())
    results = []
    
    for i in range(1, T + 1):
        A, B = map(int, input().rstrip().split())
        results.append(f"Case #{i}: {A + B}\n")
    
    sys.stdout.write(''.join(results))
    
if __name__ == "__main__":
    main()